crud.py

```diff
@@ -123,9 +110,9 @@ async def get_pay_links(wallet_ids: Union[str, List[str]]) -> List[PayLink]:
 
 
 async def update_pay_link(link_id: str, **kwargs) -> Optional[PayLink]:
-    if "lnaddress" in kwargs:
-        await check_lnaddress_format(kwargs["lnaddress"])
-        await check_lnaddress_update(kwargs["lnaddress"], link_id)
+    if len(kwargs["username"]) > 0:
+        await check_lnaddress_format(kwargs["username"])
+        await check_lnaddress_not_exists(kwargs["username"])
```

Comment:
When treating a pay link with a lnaddress call the existing `check_lnaddress_not_exists(username: str)`